app/views/admin/attachments/edit.html.erb

```diff
@@ -1,11 +1,16 @@
 <% page_title "Editing #{attachment.attachable_model_name} #{attachment.readable_type} attachment" %>
 
-<div class="row">
-  <section class="col-md-8">
-    <h1>Editing <%= attachment.attachable_model_name %> <%= attachment.readable_type %> attachment</h1>
+<div class="govuk-grid-row">
+  <section class="govuk-grid-column-two-thirds">
+    <%= render "govuk_publishing_components/components/back_link", href: attachable_attachments_path(attachable) %>
```

Comment:
If you add it like this, it will add everything you need

Reply:
```ruby
<% content_for :back_link do %>
  <%= render "govuk_publishing_components/components/back_link", {
    href: admin_edition_path(@edition)
  } %>
<% end %>
```

Reply:
Nice, have done.